core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.controller.ts

```diff
@@ -111,14 +142,18 @@ angular.module('oppia').controller(
       };
 
       $scope.acceptAndReviewNext = function() {
+        $scope.finalCommitMessage = generateCommitMessage();
+        if ($scope.isTranslationUpdated) {
+          $scope.finalCommitMessage = $scope.finalCommitMessage + '-With Edits';
+        }
```

Comment:
Can we move this logic to `generateCommitMessage` itself?

Reply:
Done